server/src/main/java/org/apache/gravitino/server/web/rest/FilesetOperations.java

```diff
@@ -110,14 +110,12 @@ public Response listFilesets(
             idents =
                 MetadataFilterHelper.filterByExpression(
                     metalake,
-                    "( (METALAKE::USE_CATALOG || CATALOG::USE_CATALOG) && "
-                        + "(METALAKE::USE_SCHEMA || CATALOG::USE_SCHEMA ||SCHEMA::USE_SCHEMA) && "
-                        + " (METALAKE::READ_FILESET || CATALOG::READ_FILESET "
-                        + "|| SCHEMA::READ_FILESET || FILESET::READ_FILESET "
-                        + "|| METALAKE::WRITE_FILESET || CATALOG::WRITE_FILESET "
-                        + "|| SCHEMA::WRITE_FILESET || FILESET::WRITE_FILESET) "
-                        + "|| METALAKE::OWNER || CATALOG::OWNER "
-                        + "|| SCHEMA::OWNER || FILESET::OWNER)",
+                    "ANY(OWNER, METALAKE, CATALOG, SCHEMA, FILESET) ||"
+                        + "("
+                        + "  ANY(USE_CATALOG, METALAKE, CATALOG) && "
+                        + "  ANY(USE_SCHEMA, METALAKE, CATALOG, SCHEMA) && "
```

Comment:
ANY(USE_SCHEMA, METALAKE, CATALOG, SCHEMA) replace to ANY_USE_SCHEMA 
ANY(USE_CATALOG, METALAKE, CATALOG) replace to ANY_USE_CATALOG